src/hooks/use-last-seen-version.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect, useCallback } from 'react';
+import { getIpcApi } from 'src/lib/get-ipc-api';
+
+interface UseLastSeenVersion {
+	lastSeenVersion: string | undefined;
+	isNewVersion: boolean;
+	updateLastSeenVersion: () => Promise< void >;
+}
+
+export function useLastSeenVersion(): UseLastSeenVersion {
+	const [ lastSeenVersion, setLastSeenVersion ] = useState< string | undefined >( undefined );
+	const currentVersion = window.appGlobals.appVersion;
+
+	useEffect( () => {
+		const fetchLastSeenVersion = async () => {
+			try {
+				const version = await getIpcApi().getLastSeenVersion();
+				setLastSeenVersion( version );
+			} catch ( error ) {
+				console.error( 'Failed to get last seen version:', error );
+			}
+		};
+		fetchLastSeenVersion();
+	}, [] );
+
+	const updateLastSeenVersion = useCallback( async () => {
```

Comment:
Did you consider using recently introduced Redux stores?

Reply:
I did not consider it before, but now made the change in https://github.com/Automattic/studio/pull/1110/commits/027f0a279e18b4cbd4ba4f05bbc0e15ff797ee43.